.gitignore

```diff
@@ -26,3 +26,7 @@
 .idea
 
 vendor/
+
+.vscode
+chains/solana/tsclient/src/staging.ts
+*node_modules
```

Comment:
Btw, just because I've seen this on other branches too, have you considered adding these to your machine's global `.gitignore`?